### ISSUE TITLE
"DeprecationWarning: GaussianBlur is no longer maintained"

### ISSUE THREAD

**Issue by TimNagle-McNaughton** [NVlabs/stylegan2-ada-pytorch#140]
**Describe the bug**
Repeated depreciation warning when training is initiated:

>c:\users\tim\.conda\envs\synth\lib\site-packages\kornia\augmentation\augmentation.py:1875: DeprecationWarning: GaussianBlur is no longer maintained and will be removed from the future versions. Please use RandomGaussianBlur instead.

**To Reproduce**

1. Using kornia-0.5.5
2. Run any basic training command ("stylegan2_pytorch --data path/to/data")
3. Warning is printed a few dozen times

**Expected behavior**
Warning should not occur, seems like there's a call to "GaussianBlur" that needs to be updated.

**Screenshots**
![image](https://user-images.githubusercontent.com/52011642/125096148-19585b00-e092-11eb-9410-980c9b55882f.png)

**Desktop:**
 - OS: Windows 10
 - PyTorch version: 1.7.1
 - CUDA toolkit version: CUDA 11.1
 - NVIDIA driver version: 471.11
 - GPU: 1080 founders
 - Docker: N/A

**Additional context**
N/A

**Comment by TimNagle-McNaughton** [comment]
Running a different command causes python to hang after the first warning:

>stylegan2_pytorch --new --data ./data --mirror=1 --image-size 256 --num-train-steps 179373 --results_dir ./results/ --models_dir ./models/ --save_every 500 --evaluate_every 100 --batch-size 16 --gradient-accumulate-every 6 --network-capacity 8 --log True

**Comment by TimNagle-McNaughton** [comment]
Removing the "--log" flag allows training to continue when the warning occurs!

**Comment by TimNagle-McNaughton** [comment]
I am an idiot, and other users should beware that "pip install stylegan2-pytorch" does NOT install this repo.
